Dashboard/app/js/lib/views/maps/map-views-common-public.js

```diff
@@ -0,0 +1,205 @@
+FLOW.NavMapsView = FLOW.View.extend({
```

Comment:
This should be fine for now since the *-public is only used when the public site is requested. When we include the cdb integration for the public maps as well we'll need to unify the implementations.